app/src/main/java/com/kickstarter/libs/KSCurrency.java

```diff
@@ -67,6 +68,37 @@ public KSCurrency(final @NonNull CurrentConfigType currentConfig) {
     return NumberUtils.format(currencyOptions.value(), numberOptions);
   }
 
+  /**
+   * Returns a currency string appropriate to the user's locale and preferred currency.
+   *
+   * @param initialValue Value to display, local to the project's currency.
+   * @param project The project to use to look up currency information.
+   * @param symbol The currency symbol that should be shown next to the pledge and goal amounts.
+   */
+  public String formatWithUserPreference(final float initialValue, final @NonNull Project project, final String symbol) {
```

Comment:
The only place this method is used is in tests, so we're not actually testing what we're showing the user. I don't think we need it since we are only using 
```formatWithUserPreference(final float initialValue, final @NonNull Project project, final @NonNull RoundingMode roundingMode, final String symbol)```